### DISCUSSION TITLE
Como ajudar em projetos no github sendo iniciante?

### DISCUSSION THREAD

**Discussion by ocaioaugusto** [frontendbr/forum#1193] [Discussão]
Estou começando a desenvolver e tenho encontrado dificuldades em achar projetos para contribuir aqui no github. 

Seja fixando bugs ou incrementando coisas. Tenho o básico do front-end e quero muito contribuir, tanto pra ajudar na comunidade quanto pra eu aprender mais a como solucionar problemas reais.

Poderiam me dar dicas de como contribuir nos projetos e, como encontrar os projetos mais simples pra começar?

Obrigado!

**Comment by luizbills** [comment]
Até os projetos grandes têm issues fáceis. Recomendo você procurar nas issues dos seus projetos prediletos e ver se tem algo que você pode fazer (provavelmente vai ter). E não foque apenas em código, você pode contribuir também ajudando em traduções, correções ou atualizações na documentação, etc

um tópico simuliar #1033

**Comment by juliovedovatto** [comment]
Tem essa talk rápida sobre esse tema. Achei sensacional.

https://youtu.be/zKWsBodjvC8 (em inglês)

**Comment by thismarcoantonio** [comment]
Também tenho muitas dificuldades com isso, já li diversos artigos, até mesmo issues aqui no forum sobre esse assunto, mas nunca consigo fazer o primeiro pull request. Sempre que vejo uma ISSUE, tem uma descrição super complicada de entender e não sei o que é realmente para resolver. Porém com o [repo open-source](https://github.com/frontendbr/open-source) aqui do frontendbr pode ser mais fácil.

**Comment by felipefialho** [comment]
Amigs, faz uns meses que participei do Alura Live, e debatemos justamente esse tema. Dei várias dicas que podem ser úteis pra vocês.

[Assistam e me falem se ajudou](https://www.youtube.com/watch?v=0mi2pi4kkVk).

**Comment by thismarcoantonio** [comment]
@felipefialho Ótimo debate, ajudou bastante, github realmente é uma grande ferramenta que nos ajuda a estar conectados e que facilita o open-source. Ajudou tanto que já fiz minha primeira PR e, mesmo sendo simples foi aceita :D

**Comment by felipefialho** [comment]
@thismarcoantonio Fico feliz pra caraleo! 😄

**Comment by ocaioaugusto** [comment]
Porra, valeu demais caras! Assisti também o talk do @felipefialho lá no Alura e me ajudou muito a tirar esse meu medo, vou estudar mais um pouco de Javascript e dar o meu primeiro PR.

Outra questão que eu queria sanar, é a de meetups aqui em SP. Já fui em algumas palestras da LeWagon, mas me senti meio perdido lá no meio por não conhecer muita gente. Tem como fazer um bom networking nesses encontros? Tem como aprender algo a mais?

**Comment by felipefialho** [comment]
@caioag Mano, sinceramente o tipo de evento que mais curto é Meetup, justamente por rolar uma interação muito maior entre as pessoas. 

Tenta puxar assunto com as pessoas no coffee, geralmente vão ter um ou outro interesse em comum. No geral, faz muito bem pra carreira sim.

**Comment by droliv** [comment]
Boa noite galera, eu fundei um projeto chamado Code Solidário que é para desenvolver sites e sistemas gratuitamente para ONGs, então se estiverem interessados em colaborar, até o momento eu sou meio que iniciante, nosso site precisa de melhorias e não temos nenhuma instituição sendo atendida ainda, tudo muito no começo, mas com muita vontade pra fazer acontecer.
Vem com a gente!
https://github.com/CodeSolidario/CodeSolidario.github.io

**Comment by luizfelipehh** [comment]
@thedannyreis Parabéns pela iniciativa! 

Sou iniciante também, assim que der tempo, vou dar uma olhada e ver no que posso ajudar.

**Comment by RenatoPeres100** [comment]
@droliv, ótima iniciativa, sou muito fã de ações sociais, e já salvei o link aqui, para quando puder contribuir, possa agregar um valor a essa iniciativa.
Sou iniciante também, porém vi que tem muito conteúdo, postado aqui e vou acessa-lo e acho que vai fazer total diferença para esse meu começo também.

**Comment by eduardosoareszup** [comment]
@droliv aqui está dando erro `404`, talvez esse seja o link correto do [CodeSolidario](https://github.com/CodeSolidario/codesolidario) , o projeto ainda está de pé?

Hoje vou tentar dar uma olhada e quem sabe contribuir com o projeto 🏎 🚀
